src/igor/show.go

```diff
@@ -130,8 +132,9 @@ func runShow(_ *Command, _ []string) {
 		nodes[v] = true
 	}
 
-	// Gather a list of which nodes are down
+	// Gather a list of which nodes are down and which nodes are unreserved
 	var downNodes []string
+	var unreservedNodes []string
 	for i := igorConfig.Start; i <= igorConfig.End; i++ {
```

Comment:
You could move this code down and combine it with the loop that "compile[s] a list of unreserved nodes".